### PR TITLE
"add README"

### PR DESCRIPTION
**Pull request by octothorpus**
Please add your own name &/ github name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,29 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Project Description
 
-Things you may want to cover:
+Find my Pet is a web application which is intended to be used as a place for pet owners
```
Comment (ninalouw) [inline README.md]: Find my Pet is a web application which enables pet owners...

File: README.md [hunk 2]
```diff
@@ -1,24 +1,29 @@
...
+to report missing pets, for the public to report missing pet sightings, with the intention
```
Comment (ninalouw) [inline README.md]: and the public to report...

File: README.md [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
+of reuniting missing pets with their owners.
 
-* Ruby version
+Setup Instructions:
 
-* System dependencies
+- rails db:create db:migrate db:seed
```
Comment (ninalouw) [inline README.md]: Use backticks `` to tell markdown to format this as code.

File: README.md [hunk 4]
```diff
@@ -1,24 +1,29 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+Project Description
```
Comment (ninalouw) [inline README.md]: Use Markdown ## to format this as an H2

File: README.md [hunk 5]
```diff
@@ -1,24 +1,29 @@
...
 
-Things you may want to cover:
+Find my Pet is a web application which is intended to be used as a place for pet owners
+to report missing pets, for the public to report missing pet sightings, with the intention
+of reuniting missing pets with their owners.
 
-* Ruby version
+Setup Instructions:
```
Comment (ninalouw) [inline README.md]: Use ## to format this as an H2

File: README.md [hunk 6]
```diff
@@ -1,24 +1,29 @@
...
 
-* System dependencies
+- rails db:create db:migrate db:seed
+- bundle
 
-* Configuration
+Keys required:
```
Comment (ninalouw) [inline README.md]: add ## to format this as h2

File: README.md [hunk 7]
```diff
@@ -1,24 +1,29 @@
...
 
-* Database creation
+ENV["email_user_name"] = "..."
```
Comment (ninalouw) [inline README.md:23]: Use `` to format this as code

File: README.md [hunk 8]
```diff
@@ -1,24 +1,29 @@
...
-* Database initialization
+ENV['FACEBOOK_APP_ID'] = "..."
+ENV['FACEBOOK_APP_SECRET'] = "..."
 
-* How to run the test suite
+ENV['aws_access_key_id'] = "..."
+ENV['aws_access_secret'] = "..."
 
-* Services (job queues, cache servers, search engines, etc.)
+List Of Contributors:
```
Comment (ninalouw) [inline README.md]: add ## to format this as an H2

**Review by ninalouw — CHANGES_REQUESTED** [review]
Please look into markdown syntax and use this to format this README correctly. Perhaps look at another README as an example of how to lay this all out.

**Review by ninalouw — COMMENTED** [review]
Could you add a link to the heroku version and say:
View the live version of the application here: `link`

**Comment by octothorpus** [comment]
@ninalouw I made the changes you requested. Thank you!

**Comment by whilestevego** [comment]
@ninalouw can you check if this is ready for approval?

**Review by ninalouw — COMMENTED** [review]
Looks good :)

**Review by ninalouw — APPROVED** [review]

**Pull request merged**